org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -498,7 +504,8 @@ public boolean replace(U bean, String queueName) throws EventException {
 
 	}
 
-	protected static final long TWO_DAYS = 48*60*60*1000; // ms
+	protected static final long TWO_DAYS = TimeUnit.DAYS.toMillis(2);
+//			48*60*60*1000; // ms
 	protected static final long A_WEEK   = 7*24*60*60*1000; // ms
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)